spec/src/modules/recommendations.js

```diff
@@ -126,7 +127,10 @@ describe('ConstructorIO - Recommendations', () => {
         fetch: fetchSpy,
       });
 
-      recommendations.getAlternativeItems(itemId, { results }).then((res) => {
+      recommendations.getRecommendations(placement, {
+        itemIds,
+        results,
```

Comment:
saw this and was immediately confused by `results`, had to look at the code to remember that it was `numResults`...

we had a discussion on this before but i don't remember why we didn't go with `numResults` 🤔

Reply:
Good point.  We should change to `numResults`.

Reply:
👍 I'll open a separate PR to change this for autocomplete as well.